doc/source/security/encrypting-threads.rst

```diff
@@ -93,4 +93,16 @@ SQL statement.
 
     :ref:`using-keyring-plugin`
 
+.. rubric:: Upgrading from version 5.7 
+
+When upgrading from version 5.7, before you turn on encryption, you should
+consult INFORMATION_SCHEMA_ENCRYPTION to verify which tables are excluded. If
+you wish to exclude a table, use ``ALTER TABLE ... ENCRYPTION='N'``. 
```

Comment:
You have to explicitly set ENCRYPTION='N' for tablespace to be excluded from encryption by encryption threads.

Reply:
Updated.